content/_data/changelogs/weekly.yml

```
@@ -11293,6 +11293,47 @@
   # pull: 5535 (PR title: Bump commons-io from 2.8.0 to 2.9.0)
   # pull: 5546 (PR title: Bump checkstyle from 8.42 to 8.43)
 
+- version: "2.297"
+  date: 2021-06-08
+  changes:
+    - type: bug
+      category: bug
+      pull: 5550
+      issue: 65766
+      authors:
+        - jglick
+      message: |-
+        A race condition in class loading could result in a <code>LinkageError</code>.
+    - type: bug
+      category: bug
+      pull: 5543
+      issue: 65751
+      authors:
+        - sratz
+      message: |-
+        Do not change fonts when artifacts are as shown as a tree.
+    - type: rfe
+      category: developer
+      pull: 5320
+      authors:
+        - olamy
+      message: |-
+        Developer: Remove commons-digester wrapper from core and the dependencies entries as well.
+    - type: TODO
+      category: TODO
+      pull: 5547
+      authors:
+        - dependabot[bot]
+        - timja
+      message: |-
+        Bump sshd-core from 2.5.1 to 2.7.0 in Jenkins CLI.
```

Now I see what you were asking.  This needs to be an `rfe`, not a `TODO`.  I think it also needs to be moved earlier in the list so that  is before the `Developer` entry rather than after the `Developer` wntry

```suggestion
    - type: rfe
      category: rfe
      pull: 5547
      authors:
        - dependabot[bot]
        - timja
      message: |-
        Bump sshd-core from 2.5.1 to 2.7.0 in Jenkins CLI.
```